test.js

```diff
@@ -33,6 +33,38 @@ describe('write', function() {
     });
   });
 
+  describe('End New Line', () => {
```

Comment:
please use `function()` to be consistent with the rest of the code. We would have to do a major bump and update travis just to get the tests to pass with fat arrows. I'll do that soon, but for now it's easier if we just keep it consistent.

Reply:
Sorry, the habit. Do you want me to configure Babel (in another PR) to allow using ES6?